### ISSUE TITLE
"dependencies" error 

### ISSUE THREAD

**Issue by otheus** [nibalizer/metadata-json-lint#11]
The dependencies code needs some "relaxing". Even --no-strict-dependency had NO EFFECT on these.

    {
     "dependencies": []
    }

Should be fine, indicating no dependencies known at present. But it doesn't work.
Similarly, only the *name* attribute of an entry should be required. The version information should be optional. 
```
     "issues_url": "",
     "dependencies": [
        { "name": "puppetlabs/stdlib" }
     ]
    }

Similarly, if the "version_requirement" tag is specified, its contents could/might be empty.
```
      "issues_url": "",
      "dependencies": [
        { "name": "puppetlabs/stdlib", "version_requirement": "" }
      ]
    }
also doesn't work. 

Work-around: add a "0" to the version_requirement

**Comment by nibalizer** [comment]
@electrical whatcha think?

**Comment by nibalizer** [comment]
So I've gone ahead and reverted #9 and released 0.0.6 to rubygems. Hopefully that will fix things. I think the feature is cool but there are a number of problems that came out of it which weren't getting fixed.
